tests/webdriver-e2e/page-objects/common.page.js

```diff
@@ -0,0 +1,18 @@
+class CommonPage {
+  get hamburgerMenu() { return $('#header-dropdown-link'); }
+  get logoutButton() { return $('.fa-power-off'); }
+  get modalBody() { return $('div.modal-body'); }
+  get yesButton() { return $('a.btn.submit.btn-danger'); }
```

Comment:
Seriously? This is how selectors work? Why did we use Protractor in the 1st place?